atom/browser/BUILD.gn

```diff
@@ -339,6 +339,11 @@ source_set("importer") {
     "//electron/build:electron_config"
   ]
 
+  include_dirs = [
+    # make sure chromium_src comes before the chrome src root
+    "//electron/chromium_src",
```

Comment:
It is used for the `BrowserProcessImpl`
```
../../electron/atom/browser/importer/profile_writer.cc:34:58: error: no member named 'app' in 'BrowserProcessImpl'
    static_cast<BrowserProcessImpl*>(g_browser_process)->app();
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  ^
```
But if I use dep `//electron/chromium_src:browser`, it will cause dep cycles
```
ERROR Dependency cycle:
  //electron/chromium_src:browser ->
  //electron/atom/browser:browser ->
  //electron/atom/browser:importer ->
  //electron/chromium_src:browser
```